assets/compat.ini

```diff
@@ -1655,3 +1655,14 @@ ULJM06024 = true
 # but apparently it does work around the timing error. See issue #15304 .
 NPUG30038 = true
 NPEG00036 = true
+
+# Doko Demo Issho,load savedata error see #18420
+UCJS10002 = true
+UCJS18002 = true
+
+# Driver '76 ,see #16904 #12054
+ULUS-10235 = true
+ULES-00740 = true
+ULES-00740/P = true

```

Comment:
This /P thingy won't actually do anything and the ID is the same as above, so might as well delete it.

Reply:
Also, please rmeove the dashes from the Driver IDs.

Did you test it with this?

Reply:
Looks like redump also have the ID with /P not sure whether it's a mistake or not http://redump.org/disc/7485/

Reply:
I asked my friend about  Platinum version, he answered me that ppsspp do not show /p in the gameid